config/samples/ovs_v1beta1_ovs.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: ovs.openstack.org/v1beta1
+kind: OVS
+metadata:
+  name: openvswitch
+spec:
+  replicas: 1
+  ovsContainerImage: "quay.io/skaplons/ovs:latest"
+  ovnContainerImage: "quay.io/tripleovictoria/openstack-ovn-controller:current-tripleo"
+  external-ids:
+    system-id: "random"
+    hostname: "devcontainer"
+    ovn-bridge: "br-int"
+    ovn-remote: "tcp:1.2.3.4:6442"
+    ovn-encap-type: "geneve,vxlan"
+    ovn-encap-ip: "10.20.30.40"
+    enable-chassis-as-gateway: false
```

Comment:
aren't we looking to keep these pods act as gateway?

Reply:
I will change that to be true by default but I don't want to remove that setting as it may be valuable in some cases to change it to false